src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -163,7 +151,7 @@ const WorkspaceInitialPage = ({
                                 ]}
                                 onPress={openEditor}
                             >
-                                <Tooltip text={policy.name}>
+                                <Tooltip text={policy.name || ''}>
```

Comment:
For this one, maybe it would be better to not render the tooltip if there is no name?

Reply:
Updated!